zyngine/zynthian_layer.py

```diff
@@ -1,13 +1,13 @@
 # -*- coding: utf-8 -*-
 #******************************************************************************
 # ZYNTHIAN PROJECT: Zynthian Layer (zynthian_layer)
-# 
+#
```

Comment:
Please, remove all "code formating" changes from the commit.
If you want to improve code format, then make a separated commit with the "code formatting" changes.